apps/server/src/shared/domain/entity/school.entity.ts

```diff
@@ -10,22 +10,13 @@ import {
 	Property,
 } from '@mikro-orm/core';
 import { UserLoginMigrationEntity } from '@shared/domain/entity/user-login-migration.entity';
-import { BaseEntity } from './base.entity';
-import { FederalStateEntity } from './federal-state.entity';
+import { SchoolFeature, SchoolPurpose } from '@src/modules/school/domain/type';
```

Comment:
Der Import ist schon ein Problem. Theoretisch wäre es besser die SchoolEntity zu verschieben in das school module. Allerdings hat diese unglaublich viele Referencen. Daher wäre es aktuell technisch ungefährlicher das SchoolFeature, SchoolPurpose und FileStorageType hier zu platzieren, Evtl. als Block mit einen Kommentar und Linie markiert das es fachlich zur school modul domain Schicht gehört.

Reply:
Ich hab die beiden Typen in @shared/domain/types verschoben.